### PR TITLE
eliminate references to 1.10 kubekins image which does not exist anymore

### PR DESCRIPTION
**Pull request by krzyzacy**
also the generated node job has screwed up versions
this should help https://github.com/kubernetes/test-infra/issues/11594#issuecomment-469484840

/shrug
/assign @amwat 
cc @yguo0905

### CODE REVIEW

**Comment by imkin** [comment]
How do we fix this? Is this a problem with the generate_tests.py ?

File: experiment/test_config.yaml [hunk 1]
```diff
@@ -1355,18 +1355,14 @@ nodeK8sVersions:
...
     prowImage: gcr.io/k8s-testimages/kubekins-e2e:v20190301-76bc03340-1.12
-  stable2:
+  stable3:
     args:
     - --repo=k8s.io/kubernetes=release-1.11
     prowImage: gcr.io/k8s-testimages/kubekins-e2e:v20190301-76bc03340-1.11
-  stable3:
-    args:
-    - --repo=k8s.io/kubernetes=release-1.10
-    prowImage: gcr.io/k8s-testimages/kubekins-e2e:v20190301-76bc03340-1.10
```
Comment (krzyzacy) [inline experiment/test_config.yaml]: @imkin here we have two `stable1` keys which is wrong..

**Comment by amwat** [comment]
Thanks!
/lgtm
Yeah, I remember removing these two from the 1.10 change because we didn't want to touch these jobs for release purposes but forgot to unpin the image.

**Pull request merged**